app/build.gradle

```diff
@@ -64,4 +64,8 @@ android {
         disable 'ExtraTranslation'
         abortOnError false
     }
+
+    configurations.all {
+        resolutionStrategy.force 'com.android.support:support-annotations:25.2.0'
+    }
```

Comment:
I suggest leaving a comment ("FIXME" or "TODO") saying this is temporary or pointing to the issue.